pkg/config/polling_manager.go

```diff
@@ -182,48 +197,58 @@ func (cm *PollingProjectConfigManager) Start(ctx context.Context) {
 	}
 }
 
-// NewPollingProjectConfigManager returns an instance of the polling config manager with the customized configuration
-func NewPollingProjectConfigManager(sdkKey string, pollingMangerOptions ...OptionFunc) *PollingProjectConfigManager {
+func (cm *PollingProjectConfigManager) setDatafileAccessTokenRequest() {
+	if cm.datafileAccessToken != "" {
+		headers := []utils.Header{{Name: "Content-Type", Value: "application/json"}, {Name: "Accept", Value: "application/json"}}
+		headers = append(headers, utils.Header{Name: "Authorization", Value: "Bearer " + cm.datafileAccessToken})
+		cm.requester = utils.NewHTTPRequester(logging.GetLogger(cm.sdkKey, "HTTPRequester"), utils.Headers(headers...))
+	}
+}
 
+func newConfigManager(sdkKey string, logger logging.OptimizelyLogProducer, configOptions ...OptionFunc) *PollingProjectConfigManager {
```

Comment:
Nit: consider putting `newConfigManager` in its own file. It is not specific to polling manager.

Reply:
it is kind of related to polling config manager, because newConfigManager is constructing polling config manager object that can be safely used by static config manager. It is just a helper method, getting all common things together